filters/shedder/admission.go

```diff
@@ -423,6 +423,8 @@ func (ac *admissionControl) Request(ctx filters.FilterContext) {
 		ctx.Serve(&http.Response{
 			StatusCode: http.StatusServiceUnavailable,
 		})
+	} else {
+		ac.metrics.IncCounter(counterPrefix + "ok." + ac.metricSuffix)
```

Comment:
I am not sure but maybe we should count `total` and `rejected`?

Reply:
changed